app/views/users/circuitverse/search.html.erb

```diff
@@ -14,10 +14,8 @@
     <% @results.map { |user| ProfileDecorator.new(user) }.each do |profile| %>
       <div class="row center-row search-user-container">
         <div class="col-xs-12 col-sm-12 col-md-5 col-lg-5">
-          <div class="row center-row">
-            <%= image_tag user_profile_picture(profile.profile_picture), alt: "project.name", class: "center-row search-usersearch-image" %>
-            <p class=" center-row search-username"><%= profile.name %></p>
-          </div>
+          <%= image_tag user_profile_picture(profile.profile_picture), alt: "project.name", class: "center-row search-usersearch-image" %>
+          <p class=" center-row search-username"><%= profile.name %></p>
```

Comment:
Better to place under div, we can give class; `flex flex-col` so that it appears in expected way